datumaro/components/format_detection.py

```diff
@@ -125,6 +139,48 @@ def require_file(self, pattern: str) -> str:
 
         self.fail(requirement_str)
 
+    @contextlib.contextmanager
+    def probe_text_file(
+        self, path: str, requirement_str: str,
```

Comment:
```suggestion
        self, path: str, requirement_desc: str,
```

Reply:
Yeah, that would be more descriptive. Updated.